src/pages/GoverningBody/GoverningBody/GoverningBody.tsx

```diff
@@ -446,7 +446,7 @@ const GoverningBody = () => {
               <Button
                 type="primary"
                 className="governingBodyInfoButton"
```

Comment:
if this button lead us to all Announcements page, so change the className as well, please
`className="getAllAnnouncementsButton"`(name can be different, it's just for example

Reply:
It has to be this one because it`s one css class